LibAddonMenu-2.0/LibAddonMenu-2.0/controls/description.lua

```diff
@@ -3,16 +3,38 @@
     text = "My description text to display.", -- or string id or function returning a string
     title = "My Title", -- or string id or function returning a string (optional)
     width = "full", --or "half" (optional)
+    disabledDesc = function() return db.someBooleanSetting end, --or boolean (optional)

```

Comment:
Please rename the property to "disabled". I only chose disabledLabel for the one in the submenu since we already added the regular "disabled" property, but that's not the case here.